test/gui/tst_vfs/test.feature

```diff
@@ -9,12 +9,10 @@ Feature: Enable/disable virtual file support
         Given user "Alice" has been created on the server with default attributes and without skeleton files
         And user "Alice" has set up a client with default settings
         When the user enables virtual file support
-        Then the "Disable virtual file support..." button should be available
+        Then VFS enabled baseline image should match the default screenshot
 
 
     Scenario: Disable VFS
         Given user "Alice" has been created on the server with default attributes and without skeleton files
         And user "Alice" has set up a client with default settings
-        And the user has enabled virtual file support
-        When the user disables virtual file support
-        Then the "Enable virtual file support (experimental)..." button should be available
```

Comment:
So can we also remove the respective step definitions?

Reply:
removed.